File: SuperMessenger/SuperMessenger/react-client/src/containers/Models/MessageFileModel.js
export default class MessageFileModel { 
  constructor(id, name, contentName, sendDate, groupId, user, isConfirmed) {
    this.id = id;
    this.name = name;
    this.contentName = contentName;
    this.sendDate = sendDate;
    this.groupId = groupId;
    this.user = user;
    this.isConfirmed = isConfirmed;
  }
}

File: SuperMessenger/SuperMessenger/Models/MessageFileModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace SuperMessenger.Models
{
    public class MessageFileModel
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string ContentName { get; set; }
        public DateTime SendDate { get; set; }

        public Guid GroupId { get; set; }
        public SimpleUserModel User { get; set; }
    }
}
